### ISSUE TITLE
wrong version number in deprecation in twisted/enterprise/adbapi.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3357]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3357|
|Type|defect|
|Created|2008-07-15 14:06:33Z|

`_unreleasedVersion` says Twisted 2.6.0, which never existed.  It meant to say Twisted 8.0.0.

<details><summary>Searchable metadata</summary>

```
trac-id__3357 3357
type__defect defect
reporter__exarkun exarkun
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1216130793000000 1216130793000000
changetime__1216303445000000 1216303445000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

```
Index: twisted/enterprise/adbapi.py
===================================================================
--- twisted/enterprise/adbapi.py        (revision 24275)
+++ twisted/enterprise/adbapi.py        (working copy)
@@ -415,7 +415,7 @@


 # Common deprecation decorator used for all deprecations.
-_unreleasedVersion = Version("Twisted", 2, 6, 0)
+_unreleasedVersion = Version("Twisted", 8, 0, 0)
 _unreleasedDeprecation = deprecated(_unreleasedVersion)


```

For some reason, I didn't have to edit any tests.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set owner to @exarkun|
|-|-|

Please apply.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [24281]) Correct deprecation version number for `twisted.enterprise.adbapi.safe`

Author: exarkun
Reviewer: therve
Fixes: [#3357](https://github.com/twisted/twisted/issues/3357)

In preparation for the 8.0 release, a deprecation version number
which was incorrectly set to 2.6 before the new release numbering
scheme was decided upon was missed.  This corrects that oversight.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
